docs/source/examples/smoothing.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
<p>That's great this just works -- I didn't expect that for some reason. It would be nice if you also added a plot below it without the smoothing (on the same colorbar) to show that temporal smoothing increases predictability.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>